Jenkinsfile

```diff
@@ -1510,10 +1520,11 @@ pipeline {
                                        inst_repos: el7_daos_repos,
                                        inst_rpms: 'environment-modules ' +
                                                   'clamav clamav-devel'
+                        sh script: get_lmd_script,
+                                   label: 'Downloading Linux Malware Detect'
                         catchError(stageResult: 'UNSTABLE', buildResult: 'SUCCESS') {
                             runTest script: "${rpm_test_pre}" +
                                             "sudo yum -y install daos-client-${daos_packages_version}\n" +
-                                            "sudo yum -y history rollback last-1\n" +
                                             "sudo yum -y install daos-server-${daos_packages_version}\n" +
```

Comment:
I think the _server_ implies (i.e. requires) the _client_.  In any case, given the high expense of `yum` transactions and the repo databases, it would be best to just combine installing all of these into a single `yum install`.

Reply:
Added to current pushed commit.